<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OperatorWidget</class>
 <widget class="QWidget" name="OperatorWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>522</width>
    <height>224</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="operator_grid">
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <property name="spacing">
    <number>6</number>
   </property>
   <item row="2" column="0" colspan="4">
    <widget class="QTabWidget" name="attributes_twg">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="arguments_tab">
      <attribute name="title">
       <string>Arguments</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="funcs_opers_tab">
      <attribute name="title">
       <string>Advanced</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <property name="leftMargin">
        <number>4</number>
       </property>
       <property name="topMargin">
        <number>4</number>
       </property>
       <property name="rightMargin">
        <number>4</number>
       </property>
       <property name="bottomMargin">
        <number>4</number>
       </property>
       <property name="spacing">
        <number>6</number>
       </property>
       <item row="1" column="0">
        <widget class="QLabel" name="join_func_lbl">
         <property name="text">
          <string>Join:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="restrict_func_lbl">
         <property name="text">
          <string>Restrict:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="commutator_op_lbl">
         <property name="text">
          <string>Commutator:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="negator_op_lbl">
         <property name="text">
          <string>Negator:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>193</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="3">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>229</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="operator_func_lbl">
     <property name="text">
      <string>Operator Func.:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QCheckBox" name="hashes_chk">
     <property name="text">
      <string>HASHES</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="options_lbl">
     <property name="minimumSize">
      <size>
       <width>70</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>Options:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QCheckBox" name="merges_chk">
     <property name="text">
      <string>MERGES</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
